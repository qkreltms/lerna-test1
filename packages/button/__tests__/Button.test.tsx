import Button from '../src';

describe('Button', () => {
  test('should match snapshot and styles for default props', () => {
    expect(Button).toMatchSnapshot();
  });
});
