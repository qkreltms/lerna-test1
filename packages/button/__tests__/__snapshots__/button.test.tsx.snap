// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match snapshot and styles for default props 1`] = `[Function]`;
